pages/profiles/snigdharao.md

```diff
@@ -0,0 +1,27 @@
+ # **Snigdha Rao Madhavaram** 
+<p>
+ <img height="250" width="250" src="https://s3.amazonaws.com/mylifecylebuckhyd12/IMG_2569+(1).jpg">
+</p>
+
+Time zone : EDT, -0400
+OS : Mac osX
+BuzzWords :Python, Java, C/C++, JS, Bash, MIPS, Matlab, Linux, Postgres, MySQL, AWS
+
+As a recent graduate with a Master’s in Computer Science from California State University, Long Beach,I am passionate about coding and love to learn new tech stacks .I have  over 2 years of work experience and i love to travel
+
+### Education
+* Master’s in Computer Science, California State University, Long Beach (GPA: 3.7) 
+*  Bachelor’s in Computer Science, Jawaharlal Nehru Technological University (GPA: 4.0)
+
+### Projects
+1. [TechFish Bank ](https://goo.gl/MmY1DK)
+    Technologies : Java, Spring Boot, Angular 2, jQuery
+2. [UberTruck]( https://goo.gl/N4xqxT) 
```

Comment:
There's an extra space before `https://goo.gl/N4xqxT`, it's not consistent with the formatting of other 2 items in the list, would it be better to remove it?